src/main/java/seedu/task/model/event/Description.java

```diff
@@ -0,0 +1,55 @@
+package seedu.task.model.event;
```

Comment:
I suppose the Description class for events is exactly the same as task? Dont think we should duplicate the classes if that's the case. No point